dist/flatpak/input-leap-flatpak

```diff
@@ -1,26 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why this got removed?

Reply:
because this is a complete hack and flatpak does it correctly already

Reply:
also flatpaks are only for gui apps so running the server or client from this does not really align w/ how flatpaks are ment to be used

Reply:
Exactly what @sithlord48 says. @whot wanted it removed as he acknowledged it was a temporary hack.